tgui/packages/tgui/interfaces/AiAirlock.js

```diff
@@ -0,0 +1,160 @@
+import { Fragment } from 'inferno';
+import { useBackend } from '../backend';
+import { Button, LabeledList, Section } from '../components';
+import { Window } from '../layouts';
+
+const dangerMap = {
+  2: {
+    color: 'good',
+    localStatusText: 'Offline',
+  },
+  1: {
+    color: 'average',
+    localStatusText: 'Caution',
+  },
```

Comment:
You never use the caution status

```suggestion
  1: {
    color: 'good',
    localStatusText: 'Offline',
  },
```